src/main/io/ledstrip.c

```diff
@@ -1055,16 +1055,14 @@ bool isOverlayTypeUsed(ledOverlayId_e overlayType)
 
 void updateRequiredOverlay(void)
 {
-    for (int timID = 0; timID < timTimerCount; timID++) {
-        requiredTimerLayer[timID] = true;
-    }
-    requiredTimerLayer[timBlink] = isOverlayTypeUsed(LED_OVERLAY_BLINK);
-    requiredTimerLayer[timLarson] = isOverlayTypeUsed(LED_OVERLAY_LARSON_SCANNER);
-    requiredTimerLayer[timWarning] = isOverlayTypeUsed(LED_OVERLAY_WARNING);
+    disabledTimerMask = 0;
```

Comment:
possibly `enabledTimerMask = (1 << timTimerCount) - 1;`

Reply:
I went with the disabled approach because initializing to 0 resets all to enabled, and then the few that need to be disabled can be set to 1.  Also won't break if anybody adds additional timers (at least up to 16 total).

Reply:
`enabledTimerMask = (1 << timTimerCount) - 1;` will enable all timers (lover timTimerCount bits)

But both ways are fine ...